src/NHibernate/UserTypes/IEnhancedUserType.cs

```diff
@@ -18,9 +25,13 @@ public interface IEnhancedUserType : IUserType
 		string ObjectToSQLString(object value);
 
 		/// <summary> 
-		/// Return a string representation of this value, as it
-		/// should appear in an XML document
+		/// Return a string representation of this value. It does not need to be xml encoded.
 		/// </summary>
+		// 6.0 TODO: rename "ToString".
+		// Since 5.2
+		[Obsolete("This method was not used in a xml context, but instead just used for logs. It will be renamed " +
+			"\"ToString\" in a future version. Implement a \"string ToString(object value)\", called by your " +
+			"\"string ToXMLString(object value)\", in order to get ready for the rename.")]
 		string ToXMLString(object value);
```

Comment:
This one is less prone to incite users to code bugs: since it is used only for logs, it will not be a serious issue if the user xml-encode the value. But for consistency, renaming too.